apps/confessions-server/.env.example

```diff
@@ -0,0 +1,19 @@
+# The port on which this server runs. We recommend `3005` for convenient
+# local integration with the rest of the services that run within this
+# repository.
+PORT=3005
+
+# The confessions server needs to know where it itself can be accessed in order
+# to be able to create URLs that point to itself.
+CONFESSIONS_SERVER_URL="http://localhost:3005"
+
+ACCESS_TOKEN_SECRET=
```

Comment:
add a comment regarding what, if anything, this should be set to in development

Reply:
added